README.md

```diff
@@ -258,7 +258,11 @@ The file `common.ts` with type [`AppConfig`](src/config/types.ts) contains impor
   - `defaultLimitedApproval`: Optional flag to set the default ERC-20 approval to limited approval. For chains where gas is low, it is recommended to set this flag to true.
   - `gasToken`: Gas token name, symbol, decimals, address and logoURI. This parameter will take priority over the `tokenListOverride`.
   - `blockExplorer`: The name and URL of the block explorer to be used in the notifications and when the network is added to the injected wallet.
-  - `rpc`: The RPC url and headers of the network, used to add the network to the injected wallet and to fetch data from the chain.
+  - `rpc`: 
+    - `url`: RPC url of the network
+    - `headers`: Headers append to each request to the RPC network
+    - `batchSize`: TThe maximum number of JSON-RPC requests to send in a batch
+    - `wait`: The maximum number of milliseconds to wait before sending a batch
```

Comment:
incorrect explanation - it's not between batches - it's between sync cycles.

Reply:
This is the wagmi provider config. I copied their description